.travis.yml

```diff
@@ -0,0 +1,16 @@
+notifications:
+  email: false
+
+sudo: true
+
+language: python
+
+services:
+  - docker
+
+install:
+  - docker build -t worldpopulation .
+  - docker run -v `pwd`:/code -i -t --rm worldpopulation jupyter nbconvert world_population_analysis.ipynb
+
+script:
+  - ls -l world_population_analysis.ipynb
```

Comment:
Shall we add `yadage-validate` to at least avoid malformed `yadage` specs? It should be enough to `pip install yadage` and then calling the CLI `yadage-validate` (see [here](https://github.com/diana-hep/yadage/blob/master/.travis.yml#L14), but without `toplevel` in our case)